foreman-selinux-enable

```diff
@@ -13,15 +13,20 @@ trap "rm -rf '$TMP'" EXIT INT TERM
 for selinuxvariant in targeted
 do
   if /usr/sbin/semodule -s $selinuxvariant -l >/dev/null; then
+
+    # Remove previously defined elasticsearch_port_t
+    # (this can be removed in future release)
+    /usr/sbin/semanage port -E | \
+      grep elasticsearch_port_t | \
+      sed s/-a/-d/g | \
+      /usr/sbin/semanage -S $selinuxvariant -i -
```

Comment:
Sure, I directly added that to the semanage temp file which should work just fine.